promotion/datastore.go

```diff
@@ -874,3 +877,80 @@ func (pg *Postgres) GetSumForTransactions(orderID uuid.UUID) (decimal.Decimal, e
 
 	return sum, err
 }
+
+// ClaimInput creates a claim
```

Comment:
maybe "contains the information needed to create a claim"?

Reply:
fixed: [87b0d3c](https://github.com/brave-intl/bat-go/pull/338/commits/87b0d3cb7cdad85e68470bd1195984eae23b96db)